<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_search</class>
 <widget class="QWidget" name="Widget_search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>251</width>
     <height>151</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_searchFriend"/>
   <widget class="QWidget" name="page_addFriend"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
